File: sharedlife-frontend/src/app/models/invitation/invitation.ts
export class Invitation {
  username: string;
  idHome: string;
  homeCode: string;
  address: string;
  constructor(u: string, i: string, homeCode: string, address: string){
    this.username = u;
    this.idHome = i;
    this.homeCode = homeCode;
    this.address = address;
  }
}

File: sharedlife-backend/src/main/java/net/tfg/sharedlife/model/Invitation.java
package net.tfg.sharedlife.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode
@Entity
@Table(name = "invitations")
public class Invitation {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	
	@Column(name = "username")
	private String username;
	
	@Column(name = "idhome")
	private String idHome;
	
	@Column(name = "homeCode")
	private String homeCode;
	
}
